pkgs/applications/networking/instant-messengers/teams-for-linux/default.nix

```
@@ -15,15 +15,15 @@
 , which
 }:
 
-stdenv.mkDerivation (finalAttrs: {
+stdenv.mkDerivation (finalAttrs: rec {
```

```suggestion
stdenv.mkDerivation (finalAttrs: {
```

We are using the `finalAttrs` pattern instead of `rec { ... }`